### ISSUE TITLE
SPI Speed change

### ISSUE THREAD

**Issue by Skaptor** [Microchip-MPLAB-Harmony/mhc#26]
MHC does not "recognize" update the BRG value when changing the SPI peripheral baudrate numeric up down control. I have to manually change the "Dummy Data" in order for the change to be reflected in the generated file.

**Comment by Skaptor** [comment]
I checked with the MHCM and it shows version: v3.9.0. The issue is rather simple if I only change the SPI baud rate, the code is not generated. I have to change another parameter such as the dummy data so the "dirtyness" is set and the change is registered and generated.

![image](https://user-images.githubusercontent.com/8442251/118018280-d2a9e700-b314-11eb-87a0-f9ff7be3606f.png)


Maybe my project has a bad setting? I'm using the PIC32MZ-EF tcp/ip rtos example and the only thing I added was the SPI peripheral driver.

**Comment by vishalnxt** [comment]
Maybe my project has a bad setting? - could be. 
You may want to test this on another demo application, may be the [spi_self_loopback_interrupt](https://github.com/Microchip-MPLAB-Harmony/csp_apps_pic32mz_ef/tree/master/apps/spi/spi_self_loopback_interrupt) application and see if the issue is reproducible here as well. 
I just tried this at my end, and it seems to change the baud rate fine.

**Comment by Skaptor** [comment]
Yeah, the strange thing here is, I am using the TCP/IP example as a project base. But how can that be project related? If I had to guess the "dirty" flag is not being set when changing that specific value, that's what makes it not change when generating code.
